submissions/YuliiaDyka/frogger-game/app.js

```diff
@@ -0,0 +1,142 @@
+// Player constants
+const defaultPlayer = {
+    x: 204,
+    y: 409,
+};
+const stepSize = {
+    x: 102,
+    y: 84,
+}
+
+// Enemy constans
+const maxX = 500,
+      enemyStartX = -150,
+      affectedArea = 60,
+      numberOfEnemies = 4,
+      firstEnemyRow = 63,
+      enemyRows = [];
+
+const initialSpeed = {
+        max: 200,
+        min: 60,
+}
+
+// Game Constants
+const winningArea = {
+    borderTop: -12,
+    borderBottom: 0,
+};
+const playingArea = {
+    maxX: 408,
+    maxY: 409,
+}
+const rowHeight = 83;
+
+// enemy rows calculation
+for (let i = 0; i < numberOfEnemies; i++) {
+    enemyRows.push(firstEnemyRow + rowHeight * i);
+}
+
+// Enemy
+
+const Enemy = function(speed, x, y) {
+    this.x = x;
+    this.y = y;
+    this.speed = speed;
+    this.sprite = 'images/enemy-bug.png'; 
+
+};
+
+Enemy.prototype.update = function(dt) {
+    this.x += this.speed * dt;
+    if (this.x > maxX) {
+        this.x = enemyStartX;
+        this.speed += 1;
+    }
+    this.isCollision(player);
```

Comment:
Still a reference to a global variable.
The requirements provide a hint on how to resolve this.